src/orion/algo/tpe.py

```diff
@@ -266,89 +270,130 @@ def set_state(self, state_dict):
         self.seed_rng(state_dict["seed"])
         self.rng.set_state(state_dict["rng_state"])
 
-    def suggest(self, num=1):
+    def _n_suggested(self):
+        return len(self._trials_info)
+
+    def _n_observed(self):
```

Comment:
Seems no difference compared with the `n_observed `property in base algorithm?

Reply:
Yeah, like _n_suggested.